innopoints/models/notification.py

```diff
@@ -16,10 +18,26 @@ class NotificationType(Enum):
     manual_transaction = auto()
     project_review_status_changed = auto()
     all_feedback_in = auto()
+    added_as_moderator = auto()
     out_of_stock = auto()
     new_purchase = auto()
     project_review_requested = auto()
-    added_as_moderator = auto()
+
+
+type_to_group = {
+    NotificationType.purchase_status_changed: 'innostore',
+    NotificationType.new_arrivals: 'innostore',
+    NotificationType.claim_innopoints: 'volunteering',
+    NotificationType.application_status_changed: 'volunteering',
+    NotificationType.service: 'service',
+    NotificationType.manual_transaction: 'service',
+    NotificationType.project_review_status_changed: 'project_creation',
+    NotificationType.all_feedback_in: 'project_creation',
+    NotificationType.added_as_moderator: 'project_creation',
+    NotificationType.out_of_stock: 'administration',
+    NotificationType.new_purchase: 'administration',
+    NotificationType.project_review_requested: 'administration',
```

Comment:
How about moving these groups to an enum to make reusing it simpler in the schema and the default in account model?

Reply:
That would, in fact, make it more difficult, for we are storing the values in the JSON, and if we use `auto` with enums, we risk messing things up, and if we use actual text strings as values, I don't see much benefit

Reply:
The only benefit would be the static analyzer (linter or whatever) giving errors if you rename one the groups and forget to do the same everywhere. kind of like the purpose of other enums like NotificationType, when I used strings initially and it generated problems